lib/ClimaCorePlots/src/ClimaCorePlots.jl

```diff
@@ -13,6 +13,55 @@ import ClimaCore:
     Topologies,
     Spaces
 
+using StaticArrays
+
+RecipesBase.@recipe function f(
+    field::Fields.SpectralElementField1D;
+    interpolate = 10,
+)
+    @assert interpolate ≥ 1 "number of element quadrature points for uniform interpolation must be ≥ 1"
+
+    FT = eltype(field)
+
+    # compute the interpolated data to plot
+    space = axes(field)
+    topology = Spaces.topology(space)
+    nelems = Topologies.nlocalelems(space)
+    QS = Spaces.quadrature_style(space)
+    quad_name = Base.typename(typeof(QS)).name
+    Nq = Spaces.Quadratures.degrees_of_freedom(QS)
+    Nu = interpolate
+    coord_field = Fields.coordinate_field(space)
+
+    xcoords = zeros(nelems * Nu, 1)
+    xdata = similar(xcoords)
+    w = vec(parent(Fields.field_values(field)))
+
+    for i in 1:nelems
+        interp_idx = (Nu * (i - 1) + 1):(Nu * i)
+        weight_idx = (Nq * (i - 1) + 1):(Nq * i)
+        vertices = Topologies.vertex_coordinates(topology, i)
+        xmin, xmax = Geometry.component(vertices[1], 1),
+        Geometry.component(vertices[2], 1)
+        coords = SVector{interpolate}(range(xmin, xmax, interpolate))
+        xcoords[interp_idx] .= coords
+        ξ, _ = Spaces.Quadratures.quadrature_points(FT, QS)
+        x = FT(0.5) * (xmin + xmax) .+ FT(0.5) * (xmax - xmin) * ξ
+        xdata[interp_idx] .=
+            Spaces.Quadratures.interpolation_matrix(coords, x) * w[weight_idx]
```

Comment:
If you want to get equispaced points, an easier option is probably the whole field to `ClosedUniform` quadrature rule, then just pull out the coordinates and values

Reply:
see the slice model plot